content/en/docs/architecture/ci-operator.md

```diff
@@ -297,6 +297,8 @@ others. You should use existing releases when your testing does not depend that
 OpenShift cluster installed. Existing releases have clearer stability expectations because you control what  kind
 of release will be used: from the latest CI candidate to stable versions already released to customers.
 
+Alternatively, a job can [claim a cluster](#testing-with-a-cluster-from-a-cluster-pool) from a cluster pool. A cluster pool maintains a number of pre-installed clusters that are ready to be used by jobs, i.e., the job can start testing with the cluster without waiting the whole process of provisioning a cluster. Those clusters are with a public available version and they are not customized with the changes from a pull request which triggers the job. A job should not claim a cluster if it is for an OCP component. The targeting users of cluster claim are the jobs for testing “something on OpenShift”.
```

Comment:
```suggestion
Alternatively, a job can [claim a cluster](#testing-with-a-cluster-from-a-cluster-pool) from a cluster pool. A cluster pool maintains a number of pre-installed clusters that are ready to be used by jobs, i.e., the job can start testing with the cluster without waiting the whole process of provisioning a cluster. Those clusters are with a public available version and they are not customized with the changes from a pull request which triggers the job. A job should not claim a cluster if it is for an OCP component. The targeting users of cluster claim are the jobs for testing “something on OpenShift”.
```

Reply:
What are you suggesting? I cannot see the difference?

Reply:
Huh. I wonder how that happened.

Reply:
fixed it

Reply:
Updated.